2024/day02/solution.md

```
@@ -0,0 +1,255 @@
+## Basic Linux Commands
+
+### Listing Commands
+
+- ```bash
+  ls 
+  ``` 
+
+- #### List the subdirectories and files available in the present directory.
+
+#### Sample output image: 
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)